assets/src/components/parts/janus-video/Video.tsx

```diff
@@ -438,6 +441,7 @@ const Video: React.FC<PartComponentProps<VideoModel>> = (props) => {
       onPause={handleVideoPause}
     >
       <source src={src} />
+      {isFirefox && <source src={updatedFFSrc} />}
```

Comment:
would it be bad to leave the browser test out and just put this as an alt video for all?